### DISCUSSION TITLE
Collect conference talks featuring Heads

### DISCUSSION THREAD

**Discussion by arhabd** [linuxboot/heads#1749] [General]
(2018) [Firmware is the new Software — Trammell Hudson, Two Sigma — Platform Security Summit 2018](https://redirect.invidious.io/watch?v=ZyZfS00LZ70)

**Comment by arhabd** [comment]
todo add more talks

**Reply by arhabd** [reply]
(2024) [Heads rolling release : roles of upstream and downstream forks (Design Session)](https://redirect.invidious.io/watch?v=mAb_kHrF6SQ)
(2024) [Future of Measured Boot such as Heads (Design Session)](https://redirect.invidious.io/watch?v=ZPeidhgNBtg)

**Reply by tlaurion** [reply]
> (2024) [Heads rolling release : roles of upstream and downstream forks (Design Session)](https://redirect.invidious.io/watch?v=mAb_kHrF6SQ) (2024) [Future of Measured Boot such as Heads (Design Session)](https://redirect.invidious.io/watch?v=ZPeidhgNBtg)

those were added.

**Comment by tlaurion** [comment]
> (2018) [Firmware is the new Software — Trammell Hudson, Two Sigma — Platform Security Summit 2018](https://redirect.invidious.io/watch?v=ZyZfS00LZ70)

Added at https://github.com/linuxboot/heads-wiki/commit/3058f495e6e24de333c485bdec80241bf5d442c6
